node/overseer/overseer-gen/tests/ui/err-08-duplicate-subsystem.stderr

```diff
@@ -0,0 +1,10 @@
+error[E0599]: no method named `sub0` found for struct `OverseerBuilder<_, OverseerFieldInit<AwesomeSubSys>, OverseerFieldUninit<f64>>` in the current scope
+  --> tests/ui/err-08-duplicate-subsystem.rs:57:10
+   |
+22 | #[overlord(signal=SigSigSig, error=OverseerError, event=Event, gen=AllMessages)]
+   | -------------------------------------------------------------------------------- method `sub0` not found for this
+...
+57 |         .sub0(AwesomeSubSys::default()) // Duplicate subsystem
+   |          ^^^^-------------------------- help: remove the arguments
```

Comment:
This error message is very unfortunate. it leaks some generated inner type info.

Reply:
It seems to be better now, isn't it?

Reply:
```
error[E0599]: no method named `sub0` found for struct `OverseerBuilder<_, Init<AwesomeSubSys>, Missing<f64>>` in the current scope
```
When we are trying to set another `AwesomeSubSys` over.

Reply:
:+1: